en/middleware/micrortps_manual_code_generation.md

```diff
@@ -83,15 +64,15 @@ $ python Tools/generate_microRTPS_bridge.py -s msg/sensor_baro.msg -r msg/sensor
 
 Code is generated for the *Client*, *Agent*, *CDR serialization/deserialization* of uORB messages, and the definition of the associated RTPS messages (IDL files). 
 
-Code for the bridge can be found here:
+Code for the bridge by default in the manual generation can be found here:
```

Comment:
Thanks. 
IF the default location is  NOT editable/changeable this text should say:
> Manually generated code for the bridge can be found here:

IF the default is really a default (i.e. it is something that has a value you can change using the tool), then this text should say. 
> Manually generated code for the bridge can be found here (by default):